nixos/modules/services/backup/syncoid.nix

```diff
@@ -308,14 +305,14 @@ in
           createHome = false;
         };
       };
-      groups = mkIf (cfg.group == "syncoid") {
+      groups = lib.mkIf (cfg.group == "syncoid") {
         syncoid = { };
       };
     };
 
-    systemd.services = mapAttrs'
+    systemd.services = lib.mapAttrs'
       (name: c:
-        nameValuePair "syncoid-${escapeUnitName name}" (mkMerge [
+        lib.nameValuePair "syncoid-${lib.escapeUnitName name}" (lib.mkMerge [
```

Comment:
Not quite able to create a PR to fix right now so just leaving a note here. `escapeUnitName` is defined in this file itself, it doesn't seem to be in lib.

Reply:
https://github.com/NixOS/nixpkgs/pull/342347